app/views/users/index.html.erb

```diff
@@ -1 +1,10 @@
-<h1>Here should be a list for all the users</h1>
\ No newline at end of file
+<h1>List of all users</h1>
+<% @users.each do |user| %>
+  <div class="user">
+    <img src=<%= user.photo %> alt="user pic" />
+    <div class="title">
+      <h3><%= link_to user.name, user_path(user) %></h3>
+      <p>Number of posts: <%= user.posts_counter %></p>
+    </div>
+  </div>
```

Comment:
- [ ] Kindly follow the given design for the `users` page as this is a requirement. This means following the layout of components as is in the given wireframes

Given design
![image](https://user-images.githubusercontent.com/81932478/214704250-39c92e46-3f19-471f-adbe-bab89ef2544c.png)

Your design
![image](https://user-images.githubusercontent.com/81932478/214704391-d139c613-b482-46f8-9332-88c6d3d3544c.png)